Fix: Printing with text drawing/labels is broken

rldhont opened the pull request:
When printing a map where a drawing label has been added with `Draw > Draw Tools > Text` then printing fails with

```
Invalid Parameter">HIGHLIGHT_LABELSIZE (';') cannot be converted into a list
```

If trying the generated URL without that parameter the error is

```
"Invalid Parameter">HIGHLIGHT_LABEL_ROTATION (';') cannot be converted into a list of float
```

If removing also this this one, then the GetPrint requests works.

Fixes #5472

rldhont committed:
Fix: Printing with text drawing/labels is broken

When printing a map where a drawing label has been added with `Draw > Draw Tools > Text` then printing fails with

```
Invalid Parameter">HIGHLIGHT_LABELSIZE (';') cannot be converted into a list
```

If trying the generated URL without that parameter the error is

```
"Invalid Parameter">HIGHLIGHT_LABEL_ROTATION (';') cannot be converted into a list of float
```

If removing also this this one, then the GetPrint requests works.

Fixes #5472

gioman commented:
@timlinux @Gustry many thanks! this was/is a baffling one. Even before your fix this was _not_ reproducible with a copy of my project on _[yours](https://perf.lizmap.com/lwc_3_8_qgis_3_40/index.php/view/map?repository=mt&project=bupi)_ infrastructure, while my server it has pretty much the same setup (including py-qgis-serrver)

rldhont committed:
QGIS Server GetPrint Highlight Label Alignment

Since QGIS Server 3.28, it is possible to defined GetPrint Highlight Label ALignment with `HIGHLIGHT_LABEL_HORIZONTAL_ALIGNMENT` and `HIGHLIGHT_LABEL_VERTICAL_ALIGNMENT`.
By default, Lizmap centers the label.
